apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.Tests/SpannerConnectionTests.AbortedTransactionRetryTests.cs

```diff
@@ -0,0 +1,337 @@
+﻿// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Api.Gax.Testing;
+using Google.Cloud.Spanner.Common.V1;
+using Google.Cloud.Spanner.V1;
+using Google.Cloud.Spanner.V1.Internal.Logging;
+using Google.Cloud.Spanner.V1.Tests;
+using Grpc.Core;
+using Moq;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Google.Cloud.Spanner.Data.Tests
+{
+    public partial class SpannerConnectionTests
+    {
+        public sealed class TransactionAbortedRetryTests
+        {
+            // Combining backoff and jittering for retries
+            // when the RpcException doesn't specify a wait time.
+            // InitialDelay = 250ms, BackoffMultiplier = 2, JitterMultiplier = 2
+            // Those are all hardwire, client code can't change them.
+            private const int SecondCallLowBo = 250;
```

Comment:
Possibly expand "Bo" to "Backoff"? It took me a little while to work out what was meant.